.github/workflows/ci.generate.ts

```diff
@@ -676,6 +676,16 @@ const ci = {
           run:
             "deno run --allow-write --allow-read --allow-run=git ./tests/node_compat/runner/setup.ts --check",
         },
+        // For remote debugging only.
+        // {
+        //   name: "Setup tmate session",
+        //   if: [
+        //     "(matrix.job == 'test' || matrix.job == 'bench') &&",
+        //     "matrix.profile == 'debug' && (matrix.use_sysroot ||",
+        //     "github.repository == 'denoland/deno')",
+        //   ].join("\n"),
+        //   uses: "mxschmitt/action-tmate@v3",
+        // },
```

Comment:
Let's keep this commented out in the codebase? Quite useful for debugging CI remotely :)